src/pages/MsigUI.vue

```diff
@@ -1,52 +1,137 @@
 <template>
-	<a-row>
-		<a-col :span="10">
-			<h1>Unsigned transaction</h1>
-		</a-col>
-		<a-col :span="4"> </a-col>
-		<a-col :span="10">
-			<h1>Signed transaction:</h1>
-		</a-col>
-	</a-row>
-	<a-row>
-		<a-col :span="10">
-			<a-textarea
-				v-model:value="unsignedJson"
-				placeholder="Basic usage"
-				:rows="4"
-				allow-clear
-			/>
-		</a-col>
-		<a-col :span="4">
-			<div>
-				<a-button type="primary" @click="sign">Sign</a-button>
-			</div>
-		</a-col>
-		<a-col :span="10">
-			<a-textarea
-				v-model:value="signedJson"
-				placeholder="Basic usage"
-				:rows="4"
-			/>
-		</a-col>
-	</a-row>
+	<a-layout-content class="contentSign">
+		<a-row>
```

Comment:
usually in css, we add hyphen rather than camelcasing, and I think the existing styles follow that, so please update it